src/test/index.ts

```diff
@@ -2,11 +2,12 @@ import { runSequentially } from "../tools/runSequentially";
 
 runSequentially({
     "scriptsPaths": [
-        "resolveNodeModuleToDenoModule",
-        "types/parsedImportExportStatement",
-        "denoifyImportExportStatement",
-        "denoifySingleFile",
-        "getInstalledVersionPackageJson",
-        "replacer"
+        "resolveNodeModuleToDenoModule"
+        // "types/parsedImportExportStatement",
+        // "denoifyImportExportStatement",
+        // "denoifySingleFile",
+        // "getInstalledVersionPackageJson",
+        // "replacer",
+        // "parseParams"
```

Comment:
Let's makes the test pass before merging.

Reply:
Oh no I forgot to uncomment it